Hospital/HospitalApi/Controllers/PrescriptionController.cs

```diff
@@ -21,8 +24,11 @@ public class PrescriptionController : ControllerBase
         private readonly IUnitOfWork _unitOfWork;
         private readonly IHttpRequestSender _httpRequestSender;
         private readonly string _integrationBaseUrl;
-        public PrescriptionController(IUnitOfWork unitOfWork, IHttpRequestSender httpRequestSender)
+        private readonly IMapper _mapper;
+
+        public PrescriptionController(IUnitOfWork unitOfWork, IMapper mapper, IHttpRequestSender httpRequestSender)
         {
+            _mapper = mapper;
             _unitOfWork = unitOfWork;
             _httpRequestSender = httpRequestSender;
             _integrationBaseUrl = "https://localhost:44302/";
```

Comment:
Probably something to consider for the future, usually, production and development addresses aren't the same so this is something we would like to get from environment variables or some kind of profiling (those json files in api projects)